tests/core/config/test_config.py

```diff
@@ -80,7 +81,26 @@ def test_configure_sql_data_node(self):
             {"foo": "bar"},
             "exposed_type",
             Scope.PIPELINE,
+            timedelta(1),
+            "qux",
+        )
+        Config.configure_sql_data_node(a, b, c, d, e, f, g, h, i, j, extra_args, exposed_type, scope, vp, property=k)
+        assert len(Config.data_nodes) == 2
+
+    def test_configure_mongo_data_node(self):
+        a, b, c, d, e, f, g, h, extra_args, scope, vp, k = (
+            "foo",
+            "db_name",
+            "collection_name",
+            None,
+            "user",
+            "pwd",
+            "host",
+            "port",
+            {"foo": "bar"},
+            Scope.PIPELINE,
```

Comment:
```suggestion
            Scope.SCENARIO,
```

To avoid future merge issues